ui/src/components/AppDetailedToolbar.css

```diff
@@ -0,0 +1,8 @@
+.details-toolbar{
+    margin-left: 3% 
```

Comment:
Alignment & spacing seems a bit off when I view it, if we can style this to be more consistent with the spacing & alignment of the header, sub-header & table header;

![Screenshot from 2019-03-19 11-29-30](https://user-images.githubusercontent.com/12085451/54602846-acd96e00-4a3a-11e9-9067-9374a6f0bd94.png)

Reply:
@damienomurchu As there is an additional section to go inbetween the breadcrumb and the table I didn't put much thought into the alignment as this may be changed by @craicoverflow's work on that.

Reply:
I've made changes to align the breadcrumb with the table so resolving this one for now.